src/lib/components/ui/Button/Button.jsx

```diff
@@ -108,8 +111,8 @@ Button.propTypes = {
   block: PropTypes.bool,
   clickHandler: PropTypes.func,
   disabled: PropTypes.bool,
-  icon: PropTypes.string,
-  iconPosition: PropTypes.oneOf(['before', 'after']),
+  icon: PropTypes.node,
```

Comment:
There should be PropTypes.element. PropTypes.node is generic type that can contains everything (like Number, String, Bool, Element, Array)

Reply:
Perhaps we could make it even stricter with `PropTypes.instanceOf(Icon)`? We can always relax this constraint to `PropTypes.element` without introducing breaking change.